MUSTsegmenter/MUSTsegmenter.py

```diff
@@ -1154,31 +1151,29 @@ def getPaintedSeedPoints(self, refVolume, segmentationMethods):
     Method that retrieves the user provided painted seeds
     """
     seedCoordinates = []
-    fidLists = slicer.mrmlScene.GetNodesByClass("vtkMRMLMarkupsFiducialNode")
-    for fidList in fidLists:
-      numFids = fidList.GetNumberOfFiducials()
-      for i in range(numFids):
-        isVisible = fidList.GetNthFiducialVisibility(i)
+    for i in range(slicer.mrmlScene.GetNumberOfNodesByClass("vtkMRMLMarkupsFiducialNode")):
+      pointList = slicer.mrmlScene.GetNthNodeByClass(i, "vtkMRMLMarkupsFiducialNode")
```

Comment:
The change of
```python
fidLists = slicer.mrmlScene.GetNodesByClass("vtkMRMLMarkupsFiducialNode")
```
to
```python
for i in range(slicer.mrmlScene.GetNumberOfNodesByClass("vtkMRMLMarkupsFiducialNode")):
  pointList = slicer.mrmlScene.GetNthNodeByClass(i, "vtkMRMLMarkupsFiducialNode")
```
helps avoid the vtkDebugLeaks as observed upon closing Slicer. You had a second reference when you assigned `GetNodesByClass` to a variable that would have needed to be unregistered (see https://slicer.readthedocs.io/en/latest/developer_guide/advanced_topics.html#python-scripts-and-scripted-modules). Instead of having to remember to unregister, the methods I'm using avoid that.